src/main/java/fi/helsinki/ubipositioning/mqtt/MqttService.java

```diff
@@ -10,49 +10,64 @@
 import fi.helsinki.ubipositioning.datamodels.Observation;
 
 /**
- * MqttService.
+ * Implementation of interface to provide consumer possibility to
+ * get all the messages that have been sent to subscribed topic after start up
+ * and to publish data in messages into a another topic.
+ *
+ * @see IMqttProvider
  */
-public class MqttService implements IMqttService{
-    public static final int MAX_OBSERVATIONS = 10000;
+public class MqttService implements IMqttService {
+    /**
+     * Default buffer for storing data that comes from subscribed topic.
+     */
+    private static final int MAX_MESSAGES_TO_STORE = 10000;
+    /**
+     * Default lifetime for beacons.
+     */
     private static final int BEACON_LIFETIME = 5;
+
     private IMqttProvider provider;
     private Gson gson;
     private HashMap<String, Beacon> beacons;
     private int observationLifetime;
-
+    private int maxMessages;
 
     /**
-     * Creates MqttService.
-     * @param mqttUrl Mqtt server URL.
-     * @param subTopic Topic to subscribe.
-     * @param pubTopic Topic to publish.
-     * @param beacons initial Beacon information.
+     * Creates instance of the class.
+     *
+     * @param mqttUrl Mqtt servers URL.
+     * @param subTopic Topic to subscribe to.
+     * @param pubTopic Topic to publish messages to.
+     * @param beacons Initial Beacon information.
```

Comment:
Does it matter if we give `mqtt://example.com` or `example.com` in the mqttUrl argument? I think we could give example of the mqtt url here, if both formats are not accepted.

Also i think we should clarify that subTopic is the source of RSSI data, and pubTopic is the destination of calculated location data.